collections/_article/funding-breakdown-and-hiring-process.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Godot Funding Breakdown & Hiring Process"
+excerpt: "This article aims to shed light on the current status of Godot's funding, hiring, and our plans for sustainable funding long term."
+categories: ["news"]
+author: Godot Foundation
+image: /storage/blog/covers/funding-breakdown-and-hiring-process.webp
+date: 2023-07-11 12:00:00
+---
+
+This article aims to shed light on the current status of Godot's funding, hiring, and our plans for sustainable funding long term.
+
+## Current funding and spending
+
+### State of Godot's funding
+
+Godot is sustained completely by donations. To date, those donations have taken the form of Patreon pledges, PayPal transfers, and one-time grants. By nature, only the Patreon donations are recurring (although some choose to make recurring payments through PayPal). Even then, we can't totally rely on them as the month-to-month amount fluctuates. At the time of writing, it is just under $14,000 USD per month, but the amount we receive after fees is much less.
+
+To account for the temporary nature of donations, we have been cautious about expanding the team. In the past, we waited until we had enough funds on hand to pay for a full year contract for a person before signing a contract with them.
+
+Additionally, when the project was much smaller than it is now, Godot was receiving more funds than it could use due to difficulties in finding suitable candidates for hiring. As a result, we were cautious about expanding the team. This led to Godot carrying funds over year-to-year.
+
+In the past two years, however, the amount of contributors has grown significantly, and more individuals that met the project requirements became available for hire. This, combined with the pressing need to turn the Godot 4.0 development into a stable release, resulted in an increased number of hires and a level a spending greater than our current level of month-to-month donations.
+
+We want to keep the current group of paid contributors, but we need more funding in order to do so.
+
+This amount of hires, at the current level of funding, is not sustainable long term. Thanks to the regular income from Patreon and PayPal, we aren't going to immediately run out of funds, but we are spending more each month than we take in. Therefore, to keep the current group of paid contributors and to maintain the current development pace, the project needs to increase the amount of funding it receives.
+
+### Current and future hires
+
+The foundation currently has 10 contractors (some part-time, some full-time) for a total cost of roughly 40,000 USD per month. These people work in a variety of fields that benefit the engine such as editor UX, rendering, production, fundraising, C# integration, and GDScript development.
+
+At the time of writing the Godot project (through the SFC and the Foundation) currently has contracts with the following people:
```

Comment:
"At the time of writing ... currently" doesn't sound grammatically correct :)

Reply:
Ah, good note
```suggestion
At the time of writing, the Godot project (through the SFC and the Foundation) has contracts with the following people:
```